### ISSUE TITLE
Ledger fails to start - segmentation fault

### ISSUE THREAD

**Issue by jayesh-bhoot** [termux/termux-packages#2624]
If its of any help, my device is Moto G5 Plus running Android 7.0.

![screenshot_20180706-004504](https://user-images.githubusercontent.com/1915507/42343272-36a35668-80b6-11e8-879f-f500632f7809.png)

**Comment by tomty89** [comment]
Seems to have something to do with the unwind symbols in libgmp on arm: 
```
$ ledger
Ledger 3.1.1-20160111, the command-line accounting tool

Copyright (c) 2003-2016, John Wiegley.  All rights reserved.

This program is made available under the terms of the BSD Public License.
See LICENSE file included with the distribution for details and disclaimer.
Segmentation fault
$ ls -l $PREFIX/local/lib/*.so*
lrwxrwxrwx 1 u0_a312 u0_a312     16 Jul  6 05:44 /data/data/com.termux/files/usr/local/lib/libgmp.so -> libgmp.so.10.3.2
lrwxrwxrwx 1 u0_a312 u0_a312     16 Jul  6 05:44 /data/data/com.termux/files/usr/local/lib/libgmp.so.10 -> libgmp.so.10.3.2
-rwxr-xr-x 1 u0_a312 u0_a312 443172 Jul  6 05:44 /data/data/com.termux/files/usr/local/lib/libgmp.so.10.3.2
$ LD_LIBRARY_PATH=$PREFIX/local/lib:$PREFIX/lib ledger
Ledger 3.1.1-20160111, the command-line accounting tool

Copyright (c) 2003-2016, John Wiegley.  All rights reserved.

This program is made available under the terms of the BSD Public License.
See LICENSE file included with the distribution for details and disclaimer.
Error: No journal file was specified (please use -f)
$
```

It segfaults as it errors out on missing journal file:
```
$ ledger
Ledger 3.1.1-20160111, the command-line accounting tool

Copyright (c) 2003-2016, John Wiegley.  All rights reserved.

This program is made available under the terms of the BSD Public License.
See LICENSE file included with the distribution for details and disclaimer.
Segmentation fault
$ touch .ledger
$ ledger
Ledger 3.1.1-20160111, the command-line accounting tool

Copyright (c) 2003-2016, John Wiegley.  All rights reserved.

This program is made available under the terms of the BSD Public License.
See LICENSE file included with the distribution for details and disclaimer.
] ^D$ rm .ledger
$ touch test
$ ledger -f test
Ledger 3.1.1-20160111, the command-line accounting tool

Copyright (c) 2003-2016, John Wiegley.  All rights reserved.

This program is made available under the terms of the BSD Public License.
See LICENSE file included with the distribution for details and disclaimer.
] ^D$ rm test
$
```
Perhaps libgmp needs a rebuild?

**Comment by jayesh-bhoot** [comment]
> It segfaults as it errors out on missing journal file:

Agreed. I saw the segfault instead of a normal error and stopped, assuming ledger won't work. Today, I tried out with a sample file and it worked.

![screenshot_20180706-112404](https://user-images.githubusercontent.com/1915507/42362117-6cdb9016-810f-11e8-80eb-e51eff43d8b5.png)

> Perhaps libgmp needs a rebuild?

Unfortunately, I don't have the technical know-how to contribute to this.

All in all, while ledger does work, the segfault should be resolved.
